mkosi.default.d/debian/10-mkosi.debian

```diff
@@ -77,6 +77,7 @@ Packages=
         # Provides libasan/libubsan
         gcc
         diffutils
+        cryptsetup
```

Comment:
for debian and ubuntu cryptsetup-bin is better I think, as that's where the binaries are, cryptsetup pulls in a lot of scripts and initramfs hooks and so on

Reply:
Dropped the two commits adding packages as I'm revamping the packages configuration for mkosi in a separate PR